sizzlingstats/LogStats.cpp

```diff
@@ -275,6 +276,53 @@ void CLogStats::FireGameEvent( IGameEvent *event )
 		WriteLog( log );
 		//L 03/21/2011 - 02:35:56: "GooB<330><STEAM_0:1:23384772><Blue>" picked up item "tf_ammo_pack"
 	}
+	else if ( FStrEq( name, "arrow_impact" ) )
+	{
+		// Only count crusader's crossbow
+		if ( event->GetInt("projectileType") == 11 )
+		{
+		
+			int healer = event->GetInt( "shooter" );
+			playerInfo &pInfo1 = m_entIndexToPlayerInfo[healer];
+
+			int patient = event->GetInt( "attachedEntity" );
+			playerInfo &pInfo2 = m_entIndexToPlayerInfo[patient];
+
+
+			// Only count arrows for alive players and on the same team
+			if (pInfo1.pPlayerInfo->IsPlayer() && !pInfo1.pPlayerInfo->IsDead() 
+			 && pInfo2.pPlayerInfo->IsPlayer() && !pInfo2.pPlayerInfo->IsDead()
+			 && pInfo1.teamid == pInfo2.teamid)
+			{

```

Comment:
I would have to test.  I wanted to protect the case where the medic shot and then immediately died.  I figured I might as well check the other case.